packages/web-components/examples/stackblitz/components/masthead-l1/src/index.js

```diff
@@ -7,59 +7,432 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import '@carbon/ibmdotcom-web-components/es/components/masthead/masthead-composite.js';
-import './index.scss';
+/**
+ * the `lit-element` package is not required in `package.json`, but is included
```

Comment:
Good catch. This PR was opened before the move to StackBlitz (or at least while the DPO team was making that migration happen), but I imagine this can be removed now.